tests/modules/utils/GEN_CSV_WRITER_tester.cpp

```diff
@@ -31,8 +31,12 @@ class GEN_CSV_WRITER_TestFixture : public CFBTestFixtureBase {
       setOutputData({&mQO, &mSTATUS});
       CFBTestFixtureBase::setup("CSV_WRITER_2");
 
+#ifdef WIN32
+      char filename[] = "./GEN_CSV_WRITER.tmp";
+#else
       char filename[] = "/tmp/fileXXXXXX";
       BOOST_CHECK(mkstemp(filename));
+#endif
```

Comment:
Perhaps @azoitl can shed a light why mk(s)temp was used for this test. If it is necessary/wanted, then Windows could use mktemp for this test

Reply:
After more research, mktemp is probably no good solution. @azoitl Would using a fixed file path in the working directory be a valid solution?

Reply:
@mhansal yes this sounds for tests a much better aproach

Reply:
Fixed.